packages/thumbprint-react/components/DatePicker/types.d.ts

```diff
@@ -0,0 +1,53 @@
+export type DateIsh = Date | string | number;
```

Comment:
What's the guidance around inlining types or creating a new file?

Reply:
I've been inlining if they're only used in one place, moving to a new file as soon as they are used in two places. In this case they're needed by the main index file and the utilities.

Reply:
Makes sense